static/backbone/js/api/views/tables/datagrid.js

```diff
@@ -61,12 +61,25 @@ define(["jquery", "backbone", "backgrid"], function ($, Backbone, Backgrid) {
                         this.delegateEvents();
                         return this;
                     }
+                }),
+                Row = Backgrid.Row.extend({
+                    initialize: function (options) {
+                        this.listenTo(this.model, "change", function (model, options) {
+                            //console.log(model.changedAttributes());
+                            if (options && options.save === false) {
```

Comment:
no, if(options && !options.save) has the same semantics.  just pointing out you don't need triple eq false unless you really want to ensure that options.save is not, for example, null